src/v2i-hub/MessageReceiverPlugin/src/MessageReceiverPlugin.cpp

```diff
@@ -502,11 +459,23 @@ int MessageReceiverPlugin::Main()
 						{
 							result+=buffer; 
 						}
-					} catch (...) {
+					} catch (std::exception const & ex) {
+					
 						pclose(pipemsg); 
-						throw; 
+						SetStatus<uint>(Key_SkippedSignVerifyError, ++_skippedSignVerifyErrorResponse);
```

Comment:
Add inline comment about what this SetStatus does, and what parameter has passed in

Reply:
Will do